pages/_app.tsx

```diff
@@ -1,8 +1,24 @@
 import { AppProps } from 'next/app'
+import Script from 'next/script'
 import { FunctionComponent } from 'react'
 
 import '@styles/styles.scss'
 
-const App: FunctionComponent<AppProps> = ({ Component, pageProps }) => <Component {...pageProps} />
+const App: FunctionComponent<AppProps> = ({ Component, pageProps }) => (
+    <>
+        <Script strategy='lazyOnload'>
```

Comment:
I will create an issue to move GTM here, as that is the typical convention for Next.js projects.